ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml

```diff
@@ -178,7 +178,8 @@ tests:
   steps:
     cluster_profile: aws-arm64
     dependencies:
-      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+      RELEASE_IMAGE_LATEST: release:multi-latest
```

Comment:
What is this line for? I know the override line because that is the expected variable to reference the release image, but this RELEASE_IMAGE_LATEST is unknown to me.

Reply:
RELEASE_IMAGE_LATEST is a variable used by the ci-operator to gather oc libs and other things needed from the payload to deploy the cluster.  Unfortunately, this line is a little unnecessary (for now) because it will always pull x86 libs given that prow is deploying on amd64.  If at some point we have different arch prow deployments, this override should be necessary.